lib/express_validator.js

```diff
@@ -136,6 +136,9 @@ var expressValidator = function(options) {
 
     locations.forEach(function(location) {
       req['check' + _.capitalize(location)] = function(param, failMsg) {
+        if(_.isPlainObject(param)) {
```

Comment:
I know this is my fault for not including a jscs yet, but let's do a space after the if